tests/test-extensions/vegan-extension/src/main/java/org/mule/test/vegan/extension/VeganFidelityOperation.java

```diff
@@ -16,7 +16,8 @@
 
 public class VeganFidelityOperation {
 
-  public String tryEat(@XmlHints(allowReferences = false) Object food) {
+  public String tryEat(@XmlHints(allowReferences = false) Object food,
+                       @Optional @XmlHints(allowInlineDefinition = false) Menu<String, Integer> menu) {
```

Comment:
why is it necessary to create your own Map implementation in order to test this? Not really sure custom Map types is something we want to support